src/test/shell/bazel/external_patching_test.sh

```diff
@@ -62,6 +62,8 @@ http_archive(
   urls=["file://${EXTREPODIR}/ext.zip"],
   build_file_content="exports_files([\"foo.sh\"])",
   patches = ["//:patch_foo.sh"],
+  patch_verbose = True,
+  patch_arguments = ["--verbose", "-p0"],
```

Comment:
This test is not portable, as `--verbose` is only recognized by the GNU version of `patch`. Can you come up with a test that works also for non-GNU `patch`? E.g., you could set `patch_tool` to a script generated by the test that produces some output before calling the actual `patch` tool.